integration/playwright.config.ts

```diff
@@ -21,7 +21,7 @@ export const common: PlaywrightTestConfig = {
   workers: process.env.CI ? numAvailableWorkers : '70%',
   reporter: process.env.CI ? 'line' : 'list',
   use: {
-    trace: 'on-first-retry',
+    trace: 'retain-on-failure',
```

Comment:
We might have flaky tests that require a retry, we're not really interested in traces for that